Tests/Client/Integration Tests/ClientTests01+Channels.swift

```diff
@@ -92,17 +92,21 @@ final class ClientTests01_Channels: TestCase {
         }
         
         let subscriptionNotifications = Client.shared.subscribe { event in
-            if case .notificationAddedToChannel = event {
-                StorageHelper.shared.increment(key: .notificationAddedToChannel)
-            }
-            
-            if case .notificationMessageNew = event {
-                StorageHelper.shared.increment(key: .notificationMessageNew)
+            DispatchQueue.main.async {
```

Comment:
Why do we need this here?

Reply:
`StorageHelper` fail tests because it's not thread safe.